lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -635,6 +635,46 @@ class ConvertAtenSqueezeAllDimsOp : public ConvertAtenSqueezeOp<AtenOpT> {
   }
 };
 
+// FIXME(AG): This will eventually go into a Tosa*Utils file
+// Convert an fp32 scalar into tosa fp32 tensor.
+static LogicalResult tosaF32TensorFromTorchFloat(ConversionPatternRewriter &rewriter,
+                                        Operation *op, Value torchScalarValue,
+                                        Value &tosaTensor) {
+  double scalarValue;
+
+  if (!matchPattern(torchScalarValue, m_TorchConstantFloat(&scalarValue)))
+    return failure();
+
+  // Construct a tosa.const
+  tosaTensor = mlir::tosa::getTosaConstTensorSingleF32(rewriter, op, scalarValue);
+
+  return success();
+}
+
+template <>
+LogicalResult ConvertAtenOp<AtenPowTensorScalarOp>::matchAndRewrite(
+    AtenPowTensorScalarOp op, OpAdaptor adaptor,
+    ConversionPatternRewriter &rewriter) const {
+
+  Value self = adaptor.self();
+  auto selfTy = self.getType().template cast<RankedTensorType>();
+
+  if (!selfTy)
+    return op.emitError("Only ranked tensor types supported in TOSA Pow");
+
+  Value expTensor;
+  Value expScalar = op.exponent();
+  if (failed(tosaF32TensorFromTorchFloat(rewriter, op.getOperation(), expScalar, expTensor)))
```

Comment:
nit: those 2 lines look very long. Can you use `clang-format` for formatting?

Reply:
While I have run clang-format, the generated code is less readable than the above one liner above.